optimum/commands/export/openvino.py

```diff
@@ -272,6 +284,7 @@ def run(self):
                     "quant_method": "awq" if self.args.awq else "default",
                     "sensitivity_metric": self.args.sensitivity_metric,
                     "scale_estimation": self.args.scale_estimation,
+                    "gptq": self.args.gptq,
```

Comment:
why not use the quant_method field that already takes awq as a value ?

Reply:
Because GPTQ and AWQ are non-exclusive algorithms and can be applied simultaneously. The same for Scale Estimation.

Reply:
I would say three methods can be applied sequentially in one run. This is what we are already practicing and getting better accuracy.

Reply:
clear misunderstanding from my side as I didn't know awq/gptq should / could be combined to get better accuracy. One possibility could be to create a dedicated `quant_method` to combine these different methods @AlexKoff88 what do you think?

Reply:
Actually, there is also the third one "Scale Estimation" so that we saw that all three together can provide the best results. I understand that it is not that easy to understand this from the user side. But for now, we are ok with this flexibility as we basically provide the optimization recipes.